ztp/argocd/Containerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/openshift/origin-cli:latest
```

Comment:
If you want golang, I think it's better to use the openshift golang container base.

but in reality, I think you need golang as the build container. not in the target.
target should be as clean as possible so it's both small sized and reduce the security attack surface.

Reply:
I used origin-cli since I needed the oc /kubectl cli inside the container. @nishant-parekh is working on "golang AS BUILDER" thing, a PR will follow when ready